greenheart/simulation/greenheart_simulation.py

```diff
@@ -210,133 +434,448 @@ def simple_solver(initial_guess=0.0):
+        print(
+            "hybrid plant capacity factor: ",
+            np.sum(hopp_results["combined_hybrid_power_production_hopp"])
+            / (hopp_results["hybrid_plant"].system_capacity_kw.hybrid * 365 * 24),
+        )
+
+    steel_finance = None
+    ammonia_finance = None
 
     if use_profast:
-        lcoe, pf_lcoe = he_fin.run_profast_lcoe(greenheart_config, orbit_config, orbit_project, capex_breakdown, opex_breakdown_annual, hopp_results, incentive_option, design_scenario, verbose=verbose, show_plots=show_plots, save_plots=save_plots)    
-        lcoh_grid_only, pf_grid_only = he_fin.run_profast_grid_only(greenheart_config, orbit_config, orbit_project, electrolyzer_physics_results, capex_breakdown, opex_breakdown_annual, hopp_results, design_scenario, total_accessory_power_renewable_kw, total_accessory_power_grid_kw, verbose=verbose, show_plots=show_plots, save_plots=save_plots)
-        lcoh, pf_lcoh = he_fin.run_profast_full_plant_model(greenheart_config, orbit_config, orbit_project, electrolyzer_physics_results, capex_breakdown, opex_breakdown_annual, hopp_results, incentive_option, design_scenario, total_accessory_power_renewable_kw, total_accessory_power_grid_kw, verbose=verbose, show_plots=show_plots, save_plots=save_plots)
-    
+        lcoe, pf_lcoe = he_fin.run_profast_lcoe(
+            greenheart_config,
+            orbit_config,
+            orbit_project,
+            capex_breakdown,
+            opex_breakdown_annual,
+            hopp_results,
+            incentive_option,
+            design_scenario,
+            verbose=verbose,
+            show_plots=show_plots,
+            save_plots=save_plots,
+        )
+        lcoh_grid_only, pf_grid_only = he_fin.run_profast_grid_only(
+            greenheart_config,
+            orbit_config,
+            orbit_project,
+            electrolyzer_physics_results,
+            capex_breakdown,
+            opex_breakdown_annual,
+            hopp_results,
+            design_scenario,
+            total_accessory_power_renewable_kw,
+            total_accessory_power_grid_kw,
+            verbose=verbose,
+            show_plots=show_plots,
+            save_plots=save_plots,
+        )
+        lcoh, pf_lcoh = he_fin.run_profast_full_plant_model(
+            greenheart_config,
+            orbit_config,
+            orbit_project,
+            electrolyzer_physics_results,
+            capex_breakdown,
+            opex_breakdown_annual,
+            hopp_results,
+            incentive_option,
+            design_scenario,
+            total_accessory_power_renewable_kw,
+            total_accessory_power_grid_kw,
+            verbose=verbose,
+            show_plots=show_plots,
+            save_plots=save_plots,
+        )
+
+        if "steel" in greenheart_config:
+            if verbose:
```

Comment:
I'm inclined to say that the bulk of the code inside the `if "steel"`.... statement should be in its own function to keep the simulation code cleaner.

Reply:
I agree, I'll move it out